docs/02-guides/override-a-component.mdx

```diff
@@ -2,14 +2,189 @@
+    configuration: {
+      providers: [],
+    },
+  });
+}
+```
+
+And now theme file will be injected in your project !
+
+## Override a component
+
+Let's add the description of a `Product` to a `ProductItem` as an example of
+overriding the base theme.
+
+The original file is:
+[`node_modules/@front-commerce/theme-chocolatine/theme/modules/ProductView/ProductItem/ProductItem.js`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/theme-chocolatine/theme/modules/ProductView/ProductItem/ProductItem.jsx)
+
+:::info
+
+Please refer to the
+[Front-Commerce’s folder structure documentation page](/docs/2.x/concepts/front-commerce-folder-structure)
+to get a better understanding of how components are organized in the base theme.
+
+:::
+
+1. copy it to: `theme/ProductView/ProductItem/ProductItem.jsx`
+2. add the description after the `<ProductOverview />` component in your
+   `ProductItem` with `{props.description}`.
+
+**But you are not done yet!** The `description` information is not included in
+the GraphQL fields fetched by the application in the base theme. You will thus
+need to update the fragment related to `ProductItem`.
+
+:::info
+
+In this case, the data fetching is made using a GraphQL request. Since our
+component use data fetched by a GraphQL request, we need to update this fragment
+to add the data we want to the request.
+
+We invite you to read the
+[request data flow](/docs/remixed/concepts/a-request-data-flow) to learn more
+about how data are fetched in Front-Commerce
+
+:::
+
+The original fragment file is collocated with the original component at:
+
+1. copy the
+   [`original ProductItemFragment`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/theme-chocolatine/theme/modules/ProductView/ProductItem/ProductItemFragment.gql?ref_type=heads)
+   to the equivalent location in your extension.
+2. add the field `description` to the fragment.
+
+   ```diff title="app/theme/modules/ProductView/ProductItem/ProductItemFragment.gql"
+   fragment ProductItemFragment on Product {
+     imageUrl
+   + description
+     ...ProductOverviewFragment
+     ...ProductItemActionsFragment
+   }
+   ```
+
+3. restart the application
+
+The data will now be fetched from GraphQL every time the `ProductItem` is used
+(product listings, upsells…) and will be available for you to render it as
+wanted.
+
+### Reuse the original component
+
+Front-Commerce has been designed with small components having a single
+responsibility. **We believe that theme overrides should cover most of your use
+cases**. For some features though, it appeared that reusing the base component
+could be really useful. For instance, if you want to add a small feature or
+wrapper without changing the core feature of a component.
+
+It is possible to import a component from the
+`@front-commerce/theme-chocolatine` node module in a standard way, as you would
+import components from other libraries.
```

Comment:
```suggestion
It is possible to import a component from the
`@front-commerce/theme/` module instead of `theme`. It is similar to if you were importing components from other libraries.
```

Reply:
fixed with your suggestion